php/php.editor/src/org/netbeans/modules/php/editor/completion/DocRenderer.java

```diff
@@ -231,6 +231,7 @@ static final class PHPDocExtractor {
         // http://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_phpDocumentor.howto.pkg.html#basics.desc
         // + table (table, tr, th, td)
 
+        private static final String SPLIT_WHITE_SPACES_REGEX = "[ \t]+"; // NOI18N
```

Comment:
Again, should not this be a `Pattern`, because of performance?

Reply:
Yes, will change it.

Reply:
Fixed